onnx/helper.py

```diff
@@ -905,3 +905,7 @@ def make_training_info(algorithm, algorithm_bindings, initialization, initializa
             binding.value = v
 
     return training_info
+
+
+# For backwards compatibility
+make_sequence_value_info = make_tensor_sequence_value_info
```

Comment:
```suggestion
def make_sequence_value_info(
        name,  # type: Text
        elem_type,  # type: int
        shape,  # type: Optional[Sequence[Union[Text, int, None]]]
        doc_string="",  # type: Text
        elem_shape_denotation=None,  # type: Optional[List[Text]]
):  # type: (...) -> ValueInfoProto
    """Makes a Sequence[Tensors] ValueInfoProto based on the data type and shape."""
    print("Warning: make_sequence_value_info has been deprecated since ONNX 1.10.0.",
        "Please now use make_tensor_sequence_value_info for TensorProto sequence. ")
    make_tensor_sequence_value_info(name, elem_type, shape, doc_string, elem_shape_denotation)
```

Reply:
Python warnings might be better: https://docs.python.org/dev/library/warnings.html#module-warnings

Reply:
Deprecation module will do it all automatically: https://deprecation.readthedocs.io/en/latest/#api-documentation

Reply:
@TomWildenhain-Microsoft would prefer not to introduce another package dependency (deprecation) and will use the python warnings instead.
@jcwchen I'll update the PR - should we mark this for removal in 1.11.0 or honor the semver guarantee and hold on to it until 2.0.0?

Reply:
I personally vote for 1.11.0 since I even don't know when ONNX 2.0.0 will be and we don't want users to rely on old deprecated functions for a longer unexpected time.

Reply:
Given the lack of consensus and clarity on the ONNX API deprecation schedule/policy I updated the warning to: 
"Deprecated in ONNX v1.10.0; `onnx.helper.make_sequence_value_info` alias will be removed in an upcoming release."